examples/renderable/main.rs

```diff
@@ -446,7 +439,12 @@ fn main() -> Result<(), Error> {
     );
 
     let game_data = GameDataBuilder::default()
-        .with::<ExampleSystem>(ExampleSystem, "example_system", &[])
+        .with(
+            UiLoaderSystem::<TextureFormat, FontFormat>::default(),
+            "",
```

Comment:
Any reason why you didn't give this an identifier?

Reply:
Not really, just didn't have anything that needed to depend on it.